angling_risk_task/experiment.js

```diff
@@ -304,7 +304,7 @@ var filled_areas = [];
 
 var game_setup = "<div class = titlebox><div class = center-text>Catch N' </div></div>" +
 "<div class = lake></div>" +
-"<div class = cooler><p class = info-text><strong>Red Fish in Cooler: </strong></p></div>" +
+"<div class = cooler><p class = info-text>&nbsp&nbsp<strong>Red Fish in Cooler: </strong></p></div>" +
```

Comment:
if you ever want to add spacing, you can specify the style of the element and not have to use those funny nbsp characters, like:

      <div class ="cooler" style="padding-left:20px;">

or if you want a margin:

      <div class ="cooler" style="margin-left:20px;">